Support installation of multiarch binaries.
Define MULTIARCH_TUPLE at config time.
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -91,8 +91,8 @@ INCLUDE_DIRECTORIES(${incDir}/..)
 IF(NOT WIN32)
   # at build time you may specify the cmake variable LIB_SUFFIX to handle
   # 64-bit systems which use 'lib64'
-  INSTALL(TARGETS yajl LIBRARY DESTINATION lib${LIB_SUFFIX})
-  INSTALL(TARGETS yajl_s ARCHIVE DESTINATION lib${LIB_SUFFIX})
+  INSTALL(TARGETS yajl LIBRARY DESTINATION lib/${MULTIARCH_TUPLE})
+  INSTALL(TARGETS yajl_s ARCHIVE DESTINATION lib/${MULTIARCH_TUPLE})
   INSTALL(FILES ${PUB_HDRS} DESTINATION include/yajl)
   INSTALL(FILES ${incDir}/yajl_version.h DESTINATION include/yajl)
 ENDIF()
